dev_scripts/qa.py

```diff
@@ -160,9 +160,16 @@
 _(Applies to Linux/Windows/MacOS. For MacOS/Windows, it requires an installer
 for the new version)_
 
-Install the previous version of Dangerzone system-wide. Open the Dangerzone
-application and enable some non-default settings. Close the Dangerzone
-application and get the container image for that version. For example
+Install the previous version of Dangerzone system-wide:
+
+* For MacOS/Windows, use the version from the website.
+* For Linux, install the previous version using our [installation instructions](INSTALL.md).
+  In order to run commands as root, execute into the container as root with
+  `podman -it -u root <container ID> bash`.
```

Comment:
We're orienting the tester to log into the test environment using `qa.py` here during all the previous steps:
https://github.com/freedomofpress/dangerzone/blob/f02597aa4fe2913513fc252b2a133a8e6ec77111/dev_scripts/qa.py#L648

For this scenario, I think it makes sense to spin up a new container, so no traces of the test installation remain. Perhaps we can make that more explicit at the beginning of the scenario.

Note: In order to be able to follow `INSTALL.md` on Ubuntu 22.04, I had to run `dirmngr` in the background, otherwise the `gpg` keyserver command would error out with "No dirmngr".

Reply:
> Note: In order to be able to follow INSTALL.md on Ubuntu 22.04, I had to run dirmngr in the background, otherwise the gpg keyserver command would error out with "No dirmngr".

I have this error too actually, and I always wondered what's the case there. I think it's because we don't install `dirmngr` in the container image, due to `--no-install-recommends`, but I'll need to take a look.

Reply:
Actually no, the `dirmngr` package is installed, but it needs to run once before it creates some dirs. This looks to me as a container limitation, but I can add a note to the tester.

Reply:
> That's why I figured that uninstalling the Dangerzone package from the end-user environment is a reasonable compromise

As a tester, I'm still not entirely sure what the instructions are asking me to do in this scenario.

- Uninstall Dangerzone from the container in which it was installed using `qa.py` in the previous steps, install a previous version, and then upgrade? (That is what I did. I did have to make sure to keep the container opened via `qa.py` running, since the `qa.py` step itself uses `run`.)
- Prepare a new container using `env.py` and manually install Dangerzone in it, then upgrade it?
- Build Dangerzone on the _host_ environment without the use of containers? Is that what you mean with system-wide?
- Something else?

Reply:
Sigh, sorry for the confusion. I'm referring to the first bullet (uninstall -> install previous -> upgrade). Would it make it more clear if we changed the text to:

> * For Linux, uninstall Dangerzone via `apt-get remove dangerzone` / `dnf uninstall dangerzone`, and install the previous version using our [installation instructions](INSTALL.md) [...]

Reply:
No worries, this is a complex process and I'm happy to be a guinea pig for the docs :). I'd only maybe change it to "uninstall Dangerzone from your test environment", but otherwise LGTM.

Reply:
Great, I've incorporated this as well in 60cb83c65aedb9c267782972124f5cd6e8599124, and restructured the text a bit for clarity.